wavefront_planner: # ns

  # Topics
  occupancyGridMapTopic: "/map"
  goalTopic: "/move_base_simple/goal"
  pathTopic: "path"

  # Frame Ids
  baselinkFrame: "base_link"
  mapFrame: "map"

  # WavePropagation Parameters
  searchUnknownSpace: false # Set false to regard unknown cells as occupied
  pathUpdateRate: 20 # Hz

waypoint_generator:

  # Topics
  waypointTopic: "waypoint"

  # Waypoint Parameters
  distanceFromRobot: 5.0
